src/main/java/mops/businesslogic/RoleServiceImpl.java

```diff
@@ -41,6 +43,9 @@ public void checkWritePermission(Account account, Directory directory) throws Mo
         boolean allowedToWrite = directoryPermissions.isAllowedToWrite(userRole);
 
         if (!allowedToWrite) {
+            log.error(String.format("The user '%s' tried to write to '%s' where has no writing permissions.",
```

Comment:
String.format is not needed here.
grammar

Reply:
still string.format.

Reply:
Sorry missed that one